src/nanoemoji/colr_to_svg.py

```diff
@@ -296,10 +296,13 @@ def glyph_region(ttfont: ttLib.TTFont, glyph_name: str) -> Rect:
     """The area occupied by the glyph, NOT factoring in that Y flips.
 
     map_font_space_to_viewbox handles font +y goes up => svg +y goes down."""
+    width = ttfont["hmtx"][glyph_name][0]
+    if width == 0:
+        width = ttfont["glyf"][glyph_name].xMax
```

Comment:
hm I'm not sure about using the xMax as replacement for advance width when this is 0... Are you doing this because of the clipping of bitmaps that overflow the viewbox #402? It feels like you are special-casing the case when a glyph bounds exceed horizontally.. Ideally some more global fix would be desirable.

Reply:
Maybe it's a bug in the input but it seemed like it happened for glyf components

Reply:
Tracking resolution of this in #404